Misc/NEWS.d/next/Library/2024-09-06-11-58-17.gh-issue-123742.jSBNUv.rst

```diff
@@ -0,0 +1,2 @@
+Added ``.strip()`` to the :meth:`email.Message.get_payload`
+to remove spaces in ``content-transfer-encoding``.
```

Comment:
Please read [the relevant parts of the devguide](https://devguide.python.org/core-developers/committing/#how-to-write-a-news-entry) and reformulate your NEWS entry accordingly.

Reply:
Also, please rewrite the PR title to accurately and succinctly reflect the actual change.[^1]

[^1]: a thumbs-up reaction is sufficient feedback to indicate that you've read a review remark.